### ISSUE TITLE
Support loading the post-analysis script from the classpath in Maven plugin

### ISSUE THREAD

**Issue by reckart** [siom79/japicmp#274]
To better support the case where multiple Maven projects or modules want to make use of the japicmp-maven-plugin, it would come in handy if the plugin could load the post-processing script from the plugin classpath and not only from the file system.

E.g.

```
<plugin>              
  <groupId>com.github.siom79.japicmp</groupId>
  <artifactId>japicmp-maven-plugin</artifactId>
  <configuration>
    ...            
    <parameter>
      <!-- The post-analysis script is contained in "that-artifact-that-contains-my-post-analysis-script" -->
      <postAnalysisScript>japicmp/post-analysis-script.groovy</postAnalysisScript>
    </parameter>
    ...
  </configuration>
  <dependencies>
    <dependency>
      <groupId>my.group</groupId>
      <artifactId>that-artifact-that-contains-my-post-analysis-script<artifactId>
    </dependency>
  </dependencies>
</plugin>
```

**Comment by siom79** [comment]
Implemented and will be released with the next version.

**Comment by siom79** [comment]
Fixed: 0.15.3
